src/editor/Editor.js

```diff
@@ -114,6 +117,21 @@ define(function (require, exports, module) {
         MAX_SPACE_UNITS         = 10,
         MAX_TAB_SIZE            = 10;
 
+    var INDICATOR_TOOLTIPS = {
+        EDIT_PROVIDER: "CTRL+E for edit provider or click on this icon",
+        DOCS_PROVIDER: "CTRL+K for docs provider or click on this icon"
```

Comment:
A couple issues here:

* On Mac you need to use `CMD` vs. `CTRL`, so you can't hard code this.
* These need to be localized strings for different languages.